A matrix: 

[[0.9142 0.1281 0.9717 ... 0.3462 0.8083 0.7827]
 [0.6538 0.3525 0.8169 ... 0.7255 0.2992 0.9622]
 [0.1858 0.4734 0.8899 ... 0.8842 0.4622 0.9327]
 ...
 [0.8971 0.1916 0.8555 ... 0.7623 0.0606 0.2942]
 [0.877  0.8773 0.0417 ... 0.7544 0.763  0.3423]
 [0.539  0.963  0.8741 ... 0.7959 0.6477 0.8717]]
B matrix: 

[[0.8491 0.0304 0.3699 ... 0.4656 0.5683 0.5137]
 [0.624  0.8834 0.116  ... 0.5328 0.5117 0.939 ]
 [0.2872 0.0505 0.6108 ... 0.519  0.0082 0.7224]
 ...
 [0.4306 0.5929 0.5594 ... 0.0131 0.7128 0.4969]
 [0.9935 0.8319 0.1446 ... 0.4704 0.7892 0.0687]
 [0.2206 0.7021 0.779  ... 0.613  0.3717 0.4115]]
RPC_SYNC DONE
C transfer done
Timestamps transfer done
[[3501.1414 3433.1992 3432.3328 ... 3444.1565 3437.2021 3469.95  ]
 [3474.5938 3455.0623 3427.926  ... 3437.001  3405.3506 3468.5657]
 [3482.424  3486.511  3444.4963 ... 3428.8071 3424.1191 3482.381 ]
 ...
 [3478.5815 3457.8433 3439.1436 ... 3442.2102 3431.0557 3481.7964]
 [3459.879  3442.1787 3411.412  ... 3415.4084 3411.361  3467.8083]
 [3500.2488 3454.3389 3429.8418 ... 3440.2212 3426.3625 3491.5598]]
[[3501.1401 3433.2122 3432.33   ... 3444.1577 3437.2068 3469.943 ]
 [3474.5933 3455.0708 3427.9324 ... 3436.9849 3405.3496 3468.5637]
 [3482.4214 3486.507  3444.4873 ... 3428.8064 3424.124  3482.387 ]
 ...
 [3478.5881 3457.838  3439.156  ... 3442.2136 3431.0493 3481.7998]
 [3459.8777 3442.182  3411.4082 ... 3415.402  3411.3635 3467.816 ]
 [3500.264  3454.3389 3429.851  ... 3440.2173 3426.3682 3491.554 ]]
1402783.1552941178
Traceback (most recent call last):
  File "./run.py", line 170, in <module>
    np.testing.assert_allclose(C_expected, C, rtol=1e-05, atol=1e-06)
  File "/python/python-x86_64/lib/python3.8/site-packages/numpy/testing/_private/utils.py", line 1592, in assert_allclose
    assert_array_compare(compare, actual, desired, err_msg=str(err_msg),
  File "/cb/toolchains/buildroot/monolith-default/202403181828-238-5e1418a5/rootfs-x86_64/usr/lib/python3.8/contextlib.py", line 75, in inner
    return func(*args, **kwds)
  File "/python/python-x86_64/lib/python3.8/site-packages/numpy/testing/_private/utils.py", line 862, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Not equal to tolerance rtol=1e-05, atol=1e-06

Mismatched elements: 1 / 190440000 (5.25e-07%)
Max absolute difference: 0.03613281
Max relative difference: 1.0577616e-05
 x: array([[3501.1401, 3433.2122, 3432.33  , ..., 3444.1577, 3437.2068,
        3469.943 ],
       [3474.5933, 3455.0708, 3427.9324, ..., 3436.9849, 3405.3496,...
 y: array([[3501.1414, 3433.1992, 3432.3328, ..., 3444.1565, 3437.2021,
        3469.95  ],
       [3474.5938, 3455.0623, 3427.926 , ..., 3437.001 , 3405.3506,...
Program completed in: 00:02:50
